src/components/molecules/SponsorsCard/index.tsx

```diff
@@ -78,10 +79,10 @@ export const SponsorsCard: FC<Props> = ({ planType, logoImages }) => {
         {heading}
       </Typography>
       <Grid container spacing={{ xs: 1, md: 3 }} mx={'auto'} width={gridWidth} columns={columns}>
-        {logoImages.map(({ image, alt }, index) => {
+        {sponsors.map(({ id, name, logo }) => {
           return (
-            <Grid xs={xs} md={md} key={`${image}${alt}${index}`}>
-              <Image src={image} layout="responsive" alt={alt} quality={100} />
+            <Grid xs={xs} md={md} key={id}>
```

Comment:
```suggestion
            <Grid xs={xs} md={md} key={`${planType}-${id}-${name}`}>
```

警告が出ないようにスポンサー情報を整備するか、keyの採番ルールを変えてしまうと良さそうです

Reply:
ありがとうございます。 [5bb5bf3](https://github.com/GoCon/2023/pull/186/commits/5bb5bf3209b5144d2989f91f2dde95c9e49c4a58) にて suggestion を採用いたしました。